src/data-parser.js

```diff
@@ -105,6 +105,7 @@ const getMessage = inputMessage => {
                 simInfo = identifySim({
                     message: inputMessage,
                     simData: simData,
+                    hlChar: '*',
```

Comment:
is hlChar name significant?

Reply:
Not very indeed. It comes from hightlightChar, I'll change it.